src/test/java/uk/gov/hmcts/cmc/claimstore/services/DirectionsQuestionnaireServiceTest.java

```diff
@@ -19,33 +24,42 @@
 import java.util.Optional;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
 import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.ASSIGNING_FOR_JUDGE_DIRECTIONS;
 import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.ASSIGNING_FOR_LEGAL_ADVISOR_DIRECTIONS;
 import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.REFERRED_TO_MEDIATION;
 import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.WAITING_TRANSFER;
 import static uk.gov.hmcts.cmc.domain.models.ClaimFeatures.DQ_FLAG;
 import static uk.gov.hmcts.cmc.domain.models.ClaimFeatures.JUDGE_PILOT_FLAG;
 import static uk.gov.hmcts.cmc.domain.models.ClaimFeatures.LA_PILOT_FLAG;
-import static uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.PilotCourt.BIRMINGHAM;
-import static uk.gov.hmcts.cmc.domain.models.directionsquestionnaire.PilotCourt.MANCHESTER;
+import static uk.gov.hmcts.cmc.domain.models.ClaimState.READY_FOR_JUDGE_DIRECTIONS;
+import static uk.gov.hmcts.cmc.domain.models.ClaimState.READY_FOR_LEGAL_ADVISOR_DIRECTIONS;
+import static uk.gov.hmcts.cmc.domain.models.ClaimState.READY_FOR_TRANSFER;
 import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.NO;
 import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.YES;
 
-public class DirectionsQuestionnaireUtilsTest {
+@RunWith(MockitoJUnitRunner.class)
+public class DirectionsQuestionnaireServiceTest {
 
     private static final String NON_PILOT_COURT_NAME = "Non pilot court name";
-
-    private static final ResponseRejection CLAIMANT_REJECTION_PILOT = ResponseRejection.builder()
-        .freeMediation(NO)
-        .directionsQuestionnaire(
-            DirectionsQuestionnaire.builder()
-                .hearingLocation(HearingLocation.builder()
-                    .courtName(BIRMINGHAM.getName())
-                    .build()
+    private static final String PILOT_COURT_NAME = "birmingham";
+
+    private static final ResponseRejection CLAIMANT_REJECTION_PILOT;
+
+    static {
```

Comment:
What is the value in putting this here instead of just at the field declaration like the others?